README.md

```diff
@@ -3,22 +3,101 @@
 [![All Contributors](https://img.shields.io/badge/all_contributors-8-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
 EOS Costa Rica Developers Documentation Site.
 
 In this site you will find the resources collected in the day-to-day life of EOS Costa Rica collaborators. Feel free to collaborate, comment and suggest changes / ideas to improve the site.
 
+<img
+  alt="Guide front page"
+  src={useBaseUrl('img/guide-page.png')}
+/>;
 
-## Contributing
+## About the Project
 
-We use a Kanban-style board. That's were we prioritize the work. [Go to Project Board](https://github.com/eoscostarica/guide.eoscostarica.io/projects/1).
+This project recollect all new knowledge gotten by our workers to help and share information.
+
+## Table of Contents
+
+1. About The Project
+1. Content Table
+1. Project Purpose
+1. Version
+1. Tech Stack
+1. Development Environment
+1. File Structure
+1. License
+1. Contributing
+1. Contributors
+1. About Company
```

Comment:
About EOS Costa Rica

Reply:
That comes from template, I forgot to change that from there too, my bad, working on